src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutPaymentContext.php

```diff
@@ -189,4 +189,34 @@ public function iShouldNotSeeAndPaymentMethods(string ...$paymentMethodsNames):
             );
         }
     }
+
+    /**
+     * @Given I do not select any payment method
```

Comment:
```suggestion
     * @When I do not select any payment method
```
And it should be moved above `@Then` methods.